### ISSUE TITLE
Suggestion : Please add the option to complete the task without logging.

### ISSUE THREAD

**Issue by adivaste** [farshadz1997/Microsoft-Rewards-bot#403]
Add the option, by which one can perform all the searches and tasks without logging in. Logging in on daily basis looks suspicious, instead use the current regular browser profile to farm the rewards. We could have option to select the profile.

**Comment by AG4lyf** [comment]
there is a session flag `--session`

**Comment by Ducky443747** [comment]
Should I be using that flag to reduce suspicion or am I fine? Thanks if you respond

**Comment by AG4lyf** [comment]
you should be using the flag. 
it comes with a cost of Storage. over the time it would be using more storage. but you can once in a while delete it so the script relogins.

**Comment by KaushikDuddala** [comment]
To combat the storage issue, I have a cronjob running daily at midnight (my script runs daily at 5 am) which deletes the cache folders which doesn't force a re-login but saves storage as majority of the storage used by the flag is for cache. You can find the cache folders at the following urls, note these are all underneath your folder for all the other files and that these folders are per account.

/profiles/{account email}/Default/PC/Cache
/profiles/{account email}/Default/PC/Code Cache
/profiles/{account email}/Default/Mobile/Cache
/profiles/{account email}/Default/Mobile/Code Cache

**Comment by Ducky443747** [comment]
Ok, thanks! So I just have to clear my cache every once in a while after using the session argument and I'm fine? Also, say you use the argument to run it everyday with other arguments such as a webhook and fast, would it save those arguments and run it everyday with the extra said arguments? Hopefully I worded it well enough.

**Comment by farshadz1997** [comment]
yes it will, the only thing you need to do is leave the script to remain open

**Comment by KevinNitroG** [comment]
> 

how about create a script to delete them?

For windows create `.bat` file
```.bat
set "target_directory=Profiles"

for /d /r "%target_directory%" %%G in (*) do (
    if /i "%%~nxG"=="Cache" (
        echo Deleting folder: "%%G"
        rd /s /q "%%G"
    )
    if /i "%%~nxG"=="Code Cache" (
        echo Deleting folder: "%%G"
        rd /s /q "%%G"
    )
)
```

For linux create `.sh` file and `chmod +x [yourfile.sh]`
```.sh
#!/bin/bash

target_directory="Profiles"

find "$target_directory" -type d -name "Cache" -o -name "Code Cache" | while read -r folder; do
    echo "Deleting folder: $folder"
    rm -rf "$folder"
done
```

I used chatgpt to write these :)) lol, windows script works for me .-.
